torchtitan/experiments/README.md

```diff
@@ -0,0 +1,18 @@
+## Contributing Guidelines
+
+We provide this `experiments/` folder to host experiments that add significant value to `torchtitan`, with the following principles. We refer to the part of `torchtitan` outside `experiments` as `core`.
+1. Each subfolder in `experiments` will be an experiment, with a clear theme which can be flexible, such as
+    - a new model, or preferably a new model architecture, with its training infrastructure including parallelization functions;
+    - an enhancement or addition to the existing infrastructures in `torchtitan`.
```

Comment:
existing infrastructure of ( no s needed for  infrastructure, of instead of in).